plugins/woocommerce/tests/e2e-pw/tests/merchant/order-emails.spec.js

```diff
@@ -94,6 +97,37 @@ test.describe( 'Merchant > Order Action emails received', () => {
 		).toContainText( `[${ storeName }]: New order #${ newOrderId }` );
 	} );
 
+	test( 'can receive cancelled order email', async ( { page, baseURL } ) => {
+		const api = new wcApi( {
+			url: baseURL,
+			consumerKey: process.env.CONSUMER_KEY,
+			consumerSecret: process.env.CONSUMER_SECRET,
+			version: 'wc/v3',
+		} );
+		await api
+			.post( 'orders', {
+				status: 'processing',
+				billing: customerBilling,
+			} )
+			.then( ( response ) => {
+				cancelledOrderId = response.data.id;
+				api.put( `orders/${ cancelledOrderId }`, {
+					status: 'cancelled',
+				} );
+			} );
+		// search to narrow it down to just the messages we want
+		await page.goto(
+			`wp-admin/tools.php?page=wpml_plugin_log&s=${ encodeURIComponent(
+				customerBilling.email
+			) }`
+		);
+		await expect(
+			page.getByText(
+				`[${ storeName }]: Order #${ cancelledOrderId } has been cancelled`
+			)
+		).toBeVisible();
```

Comment:
Hi @lanej0 👋 This same approach can be used in the test above, `can receive new order email`. If this looks good to you and the team, I can also update other tests.